lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -117,17 +333,22 @@ class window.Turbolinks
   @loadPage: (url, xhr, options = {}) ->
     triggerEvent 'page:receive'
     options.updatePushState ?= true
-
-    if doc = processResponse(xhr, options.partialReplace)
+    if upstreamDocument = processResponse(xhr, options.partialReplace)
       reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
+      nodes = changePage(
```

Comment:
It's really easy to tophat this stuff.  

`cp lib/assets/javascripts/turbograft/turbolinks.coffee ~/.gem/ruby/2.2.3/gems/turbograft-0.2.3/lib/assets/javascripts/turbograft/`

Sprockets will instantly update your bundles to use your code (don't even have to restart your server!)  Please don't mess around with updating Gemfiles, or publishing versions until a basic tophat with this hacky technique passes.